### ISSUE TITLE
Should we move the primitives library to a different repository?

### ISSUE THREAD

**Issue by leluque** [leluque/webdiagrams#9]
The primitives library is an independent library. Should we move it to a different repository? I believe so. This way, I can correct the commit mistake. What do you think?

**Comment by ioxua** [comment]
I agree. But in order to organize library dependencies, I recommend turning both the projects into NPM modules
